react/features/invite/components/add-people-dialog/AbstractAddPeopleDialog.tsx

```diff
@@ -158,7 +158,7 @@ export default class AbstractAddPeopleDialog<P extends IProps, S extends IState>
 
         const { _callFlowsEnabled, dispatch } = this.props;
 
-        return dispatch(invite(invitees))
+        return dispatch(invite(invitees)) // @ts-ignore
```

Comment:
Why do we need this ts-ignore? It looks like changing the return type of the function returned by `invite` fixes the issue. Isn't it?

Reply:
Right. I missed that return type. Fixed it now